run/system-package/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use the official Python image.
+# https://hub.docker.com/_/python
+FROM python:3.7
+
+# [START run_system_package_ubuntu]
+RUN apt-get update -y && apt-get install -y \
+  graphviz \
+  && apt-get clean
```

Comment:
Bash commands should not be escaped with `\` unnecessarily (i.e. between `install` and `graphviz`).

Would this work and be more readible?

```sh
RUN apt-get update -y
RUN apt-get install -y graphviz
RUN apt-get clean
```

Reply:
It's recommended to reduce the number of separate `RUN`s where it makes sense to minimize the number of layers. https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers

Reply:
That makes sense.

However, I'd still not put `graphviz` on it's own line:

```sh
RUN apt-get update -y \
  apt-get install -y graphviz \
  apt-get clean
```

Reply:
Currently it matches the snippet from the [go sample](https://github.com/GoogleCloudPlatform/golang-samples/blob/master/run/system_package/Dockerfile).  Should I update both?

Reply:
While conciseness is nice, the reason most of the samples use a dedicated line for the package is it makes the diffs much cleaner to have one package per line for the future.

Reply:
Going to resolve this and keep it consistent with the other language samples.